features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/rolesandpermissions/RolesAndPermissionsPresenter.kt

```diff
@@ -47,14 +49,20 @@ class RolesAndPermissionsPresenter @Inject constructor(
     override fun present(): RolesAndPermissionsState {
         val coroutineScope = rememberCoroutineScope()
         val roomInfo by room.roomInfoFlow.collectAsState(initial = null)
+        val roomMembers by room.membersStateFlow.collectAsState()
+        val joinedRoomMemberIds by remember {
```

Comment:
Maybe add a comment to explain why this is needed so we don't forget in the future?